lib/elixir/lib/calendar/date.ex

```diff
@@ -757,6 +760,32 @@ defmodule Date do
     end
   end
 
+  @doc """
+  Shifts a date by given duration according to its calendar.
+
+  Check `Calendar.ISO.shift_date/4` for more information.
```

Comment:
This is the primary API, all of the docs should be here. We shouldn't point most users to `Calendar.ISO` (rather only the ones who need custom calendars).